### PR TITLE
Web html pulkit assignment 1

### PR DESCRIPTION
**Pull request by meta-pulkit-saxena**

### CODE REVIEW

File: Web/HTML Assignment-1/Question-1.html [hunk 1]
```diff
@@ -0,0 +1,176 @@
+<!DocType html>
+<html>
+	<head>
+		<Title>
```
Comment (shikhar-metacube) [inline Web/HTML Assignment-1/Question-1.html]: Always keep symmetry for html tags,all tags should be in similar case either uppercase or lowercase.

File: Web/HTML Assignment-1/Question-1.html [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+		</Title>
+	</head>
+	<Body>
+		<table align = "center">
+			<tr >
+				<td colspan ="2"><b>Create Your Account:</b></td>
+			</tr>
+			<tr>
+				<td align="right">Email ID:</td>
+				<td><input type="email" placeholder="Enter your email" ></td>
```
Comment (shikhar-metacube) [inline Web/HTML Assignment-1/Question-1.html]: There should be name attribute for all form input elements that you are submitting with form.

File: Web/HTML Assignment-1/Question-1.html [hunk 3]
```diff
@@ -0,0 +1,176 @@
+<!DocType html>
+<html>
+	<head>
+		<Title>
+			New Account
+		</Title>
+	</head>
+	<Body>
+		<table align = "center">
```
Comment (shikhar-metacube) [inline Web/HTML Assignment-1/Question-1.html]: As it is a user input form view, why form is not used here???

File: Web/HTML Assignment-1/Question-1.html [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+			</tr>
+			<tr>
+				<td align="right">Your Name:</td>
+				<td ><input type="text"></td>
+			</tr>
+			<tr>
+				<td align="right">Current Location</td>
+				<td>
+					<select name="Select"> 
+						<option>select</option>
```
Comment (shikhar-metacube) [inline Web/HTML Assignment-1/Question-1.html:38]: value attribute for select options is missing.

File: Web/HTML Assignment-1/Question-1.html [hunk 5]
```diff
@@ -0,0 +1,176 @@
...
+				<td>	<select name="Select"> 
+						<option>select</option>
+						<option>1</option> 
+						<option>2</option> 
+						<option>3</option>
+					</select> years
+						<select name="Select"> 
+						<option>select</option>
+						<option><6</option> 
+						<option>>6</option>
```
Comment (shikhar-metacube) [inline Web/HTML Assignment-1/Question-1.html]: why only 6 in months drop-down???

File: Web/HTML Assignment-1/Question-1.html (hunk 1 above)
Comment (shikhar-metacube) [inline Web/HTML Assignment-1/Question-1.html]: Code should be formatted properly.

Comment (meta-pulkit-saxena) [inline Web/HTML Assignment-1/Question-1.html]: Done.

File: Web/HTML Assignment-1/Question-1.html (hunk 3 above)
Comment (meta-pulkit-saxena) [inline Web/HTML Assignment-1/Question-1.html]: Done.

File: Web/HTML Assignment-1/Question-1.html (hunk 2 above)
Comment (meta-pulkit-saxena) [inline Web/HTML Assignment-1/Question-1.html]: Done.

File: Web/HTML Assignment-1/Question-1.html (hunk 4 above)
Comment (meta-pulkit-saxena) [inline Web/HTML Assignment-1/Question-1.html:38]: Done.

File: Web/HTML Assignment-1/Question-1.html (hunk 5 above)
Comment (meta-pulkit-saxena) [inline Web/HTML Assignment-1/Question-1.html]: Done.

File: Web/HTML Assignment-1/Question-1.html [hunk 6]
```diff
@@ -0,0 +1,193 @@
...
+				<td><input type="email" name="email"
+					placeholder="Enter your email"></td>
+			</tr>
+			<tr>
+				<td align="right">Password</td>
+				<td><input type="password" name="password"></td>
+			</tr>
+			<tr>
+				<td align="right">Confirm Password</td>
+				<td><input type="password" name="confirm-password"></td>
```
Comment (shikhar-metacube) [inline Web/HTML Assignment-1/Question-1.html:23]: name should be as per naming conventions that we should in any programming language.

**Pull request merged**